src/hotspot/cpu/x86/x86.ad

```diff
@@ -2677,28 +2534,10 @@ void vec_spill_helper(C2_MacroAssembler *masm, bool is_load,
         __ movq(as_XMMRegister(Matcher::_regEncode[reg]), Address(rsp, stack_offset));
         break;
       case Op_VecX:
-#ifndef _LP64
```

Comment:
Here and in following code you left code for 32-bit instead of 64-bits.

Reply:
Oh (facepalms). I guess the lesson here is not touch .ad very late at night :) Reverted to hopefully proper form. I look through the patch again, and I think this 4 adjacent hunks are the only place I made this mistake.